lib/commands/actions.js

```diff
@@ -166,6 +167,16 @@ commands.getScreenshot = async function () {
     await fs.unlink(localFile);
   }
   await this.adb.pull(png, localFile);
+  let screenOrientation = await this.adb.getScreenOrientation();
+  if (screenOrientation) {
+    jimp.read(localFile).then(function (image) {
```

Comment:
assuming that `jimp.read` and `jimp.write` return promises, this section should be rewritten as:

```js
try {
  let image = await jimp.read(localFile);
  await image.rotate(90 * screenOrientation).write(localFile);
} catch (err) {
  log.warn(`Could not rotate screenshot due to error: ${err}`);
}
```